ci-operator/config/openshift/sandboxed-containers-operator/openshift-sandboxed-containers-operator-master__4.9.yaml

```diff
@@ -0,0 +1,52 @@
+base_images:
+  ocp_builder_golang-1.13:
+    name: builder
+    namespace: ocp
+    tag: golang-1.13
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.13
+images:
+- from: ocp_builder_golang-1.13
+  to: sandboxed-containers-operator
+operator:
+  bundles:
+  - as: sandboxed-containers-operator-bundle
+    dockerfile_path: bundle.Dockerfile
+  substitutions:
+  - pullspec: registry-proxy.engineering.redhat.com/rh-osbs/openshift-sandboxed-containers-operator:1.0.0-5
```

Comment:
@wainersm while you are at it, can you please change the pullspec to refer to `controller:latest`